### PR TITLE
add "clone: false" option to InheritableAttr

### PR DESCRIPTION
**Pull request by katafrakt**
I'm getting `TypeError: can't clone Fixnum` (and related, like `Float`) while using `inheritable_attr`. The list of uncloneable classes is still probably not exhaustive, but I added `Numeric` to it.

### CODE REVIEW

**Comment by apotonick** [comment]
Thanks!

Maybe we should allow people to say `clone: false` instead?

**Comment by katafrakt** [comment]
Good idea! also thought about rescuing from TypeError but `clone: false` sounds like a better way.

**Comment by apotonick** [comment]
`rescue`ing will always bring trouble. When we inverse the responsibility, it's the user's fault if they miss to set it haha! :laughing: A great strategy I've used in many other places.

**Comment by apotonick** [comment]
Are you interested in implementing `callable: false`, @katafrakt ?

**Comment by katafrakt** [comment]
If you mean `clone: false`, I guess I can do it later this weekend ;)

**Comment by apotonick** [comment]
Haha, yes! :beers:

**Comment by katafrakt** [comment]
Should I keep current list of uncloneables for backwards compatibility? On one hand it would be wise not to force users to change their code, on the other - it would make the code more complicated.

**Comment by apotonick** [comment]
If you don't want to deprecate it, we can minor bump and simply change it to `:clone`. You have to wait until I merge this, then. https://github.com/apotonick/uber/issues/9

 I know how hard it is to deprecate things, and in the end you added 15 lines of "deprecation code" that affects 2% of your users. :wink: I am hearin' ya!

**Comment by katafrakt** [comment]
Ok, here is some naive (maybe) implementation. Let me know what you think. Then I can update readme and squash everything.

Also about deprecation, should I include some deprecation message when someone relies on built-in uncloneable types?

File: lib/uber/inheritable_attr.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module Uber
   module InheritableAttr
-    def inheritable_attr(name)
+
+    def inheritable_attr(name, options={})
       instance_eval %Q{
```
Comment (apotonick) [inline lib/uber/inheritable_attr.rb]: You can simply use `options` in the `InheritableAttribute.inherit_for` call, that way we don't need any additional code.

```ruby
InheritableAttribute.inherit_for(self, :#{name}, options)
```

You don't need the `_cloneability` fix then! :grimacing:

Comment (katafrakt) [inline lib/uber/inheritable_attr.rb]: Right. Guess I need some more conference exp to code efficiently after some beers ;) Anyway, I've rewritten that and pushed here.

**Pull request merged**